app.py

```diff
@@ -48,9 +64,18 @@ def choice():
 def final():
 		return render_template('finally.html')
 
-@app.route('/signup')
+@app.route('/signup', methods=['GET', 'POST'])
```

Comment:
You added 2 new methods here, GET and POST, yet you aren't filtering on either. 
Are you sure you need this?

Reply:
Yeah in this commit they don't make a lot of sense. I should have left it out for this one. I ended up adding code using those in the subbranch, [https://github.com/robertbernardbrown/Pizza/tree/subpizzalogin2]

Reply:
Yeah in this commit they don't make a lot of sense. I should have left it out for this one. I ended up adding code using those in the subbranch, subpizzalogin2 (Yeah in this commit they don't make a lot of sense. I should have left it out for this one. I ended up adding code using those in the subbranch, subpizzalogin2 (https://github.com/robertbernardbrown/Pizza/tree/subpizzalogin2)

Reply:
I think that's the case anyway, let me know if I'm wrong